pyecharts_snapshot/main.py

```diff
@@ -13,36 +12,50 @@
 else:
     from io import BytesIO
 
-PHANTOMJS_EXE = "phantomjs"
-NOT_SUPPORTED_FILE_TYPE = "Do not support file type %s"
+PHANTOMJS_EXEC = "phantomjs"
+NOT_SUPPORTED_FILE_TYPE = "Not supported file type '%s'"
 DEFAULT_DELAY = 1.5
 
 
 def main():
-    if len(sys.argv) < 2 or len(sys.argv) > 4:
-        raise sys.exit(-1)
-
-    try:
-        does_phantomjs_exist()
-    except OSError:
-        print("No phantomjs found in your path. Please install it!")
-        sys.exit(-1)
+    help = '''Usage:   snapshot {input file} {output type:[png|jpeg|gif|pdf]} {delay in seconds}
+         snapshot help: display this help message
+         snapshot help online: document online.'''
+    if len(sys.argv) < 2:
+        print('Missing required arguments! snapshot help for more information.')
+        sys.exit(0)
+    else:
+        if sys.argv[1] == 'help':
```

Comment:
please do not golden plate a single usage. 


The Zen of Python
Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex. <--- simple please!